src/main/java/by/onliner/cucumber/steps/CatalogSteps.java

```diff
@@ -0,0 +1,23 @@
+package by.onliner.cucumber.steps;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import static by.onliner.core.utils.EqualsUtil.equalContains;
+
+public class CatalogSteps extends AbstractSteps{
+
+    @When("Input {string} in search input field")
+    public void inputTextInSearchField(String text) {
+        mainPage.getMenu().inputTextInSearchField(text);
+    }
+
+
+    @Then("Searched item name will contain {string}")
+    public void searchedItemNameWillContain(String itemName) {
```

Comment:
Use proper naming for methods, method's name should start with a verb (ex. add, send, get and etc)

Reply:
Renamed to "willSearchedItemContainName"